odoo/addons/base/migrations/12.0.1.3/post-migration.py

```diff
@@ -25,7 +25,9 @@ def generate_thumbnails(env):
 def update_res_company_onboarding_company_state(env):
     # based on old base_onboarding_company_done
     good_companies = env["res.company"].with_context(
-        active_test=False).search([('street', '!=', False)])
+        active_test=False).search([]).filtered(
```

Comment:
For making it more readable and as there's no `active` field in `res.company`:
```suggestion
    good_companies = env["res.company"].search([]).filtered(lambda c: (
        c.partner_id.browse(
            c.partner_id.sudo().address_get(adr_pref=['contact'])['contact']
        ).sudo().street
    ))
```